hooks/DirectX.cpp

```diff
@@ -41,6 +41,14 @@ typedef struct Cache
 
 static cache_t s_Cache;
 
+ImVec2 DirectX::GetWindowSize()
```

Comment:
This brakes windowed mode, prob bcs of the title bar and edges of the window.

Reply:
You were spot on, I have used your suggestion